plugins/Shotgun/vrShotgun.py

```diff
@@ -14,23 +14,23 @@
 
 sgtk.LogManager().initialize_base_file_handler("tk-vred")
 logger = sgtk.LogManager.get_logger(__name__)
-vrShotGrid_form, vrShotGrid_base = uiTools.loadUiType("vrShotGridGUI.ui")
+vrShotgun_form, vrShotgun_base = uiTools.loadUiType("vrShotgunGUI.ui")
```

Comment:
Did we mean to rename these files from ShotGrid back to Shotgun?

plugins/ShotGrid/vrShotGrid.py → plugins/Shotgun/vrShotgun.py
plugins/ShotGrid/vrShotGridGUI.ui → plugins/Shotgun/vrShotgunGUI.ui
plugins/ShotGrid/vrShotGridGUI.ui → plugins/Shotgun/vrShotgunGUI.ui